stewards.json

```diff
@@ -14,5 +14,10 @@
     {
         "ca": "blockscape/server_ca.crt",
         "endpoint": "https://steward.blockscape.network:5734"
+    },
+    {
+        "ca": "chill_validation/server_ca.crt",
+        "ca_hosted": "https://steward0.chillvalidation.com/server_ca.crt",
```

Comment:
The `ca_hosted` field is a legacy from older instructions on this repo, perhaps you forked an earlier copy. It's no longer necessary, so you can remove it, and if you were using a reverse proxy to front two different services on your URL you should be able to remove the webserver and only have steward.

The certificate and steward endpoint look correct, but this can't be merged pending the above update and merge conflict resolution.

Reply:
Removed ca_hosted from the file.  Thank you Eric!